src/app/routes/connector-ui/dashboard-page/dashboard-page/dashboard-page-data.service.ts

```diff
@@ -141,8 +139,8 @@ export class DashboardPageDataService {
   ): DonutChartData {
     const filteredTransfers =
       direction === 'incoming'
-        ? transfers.filter((it) => this.transferProcessUtils.isIncoming(it))
-        : transfers.filter((it) => this.transferProcessUtils.isOutgoing(it));
+        ? transfers.filter((it) => it.direction === 'incoming')
+        : transfers.filter((it) => it.direction === 'outgoing');
```

Comment:
why was this changed? the first util functions seems more readable and help by code reuse

Reply:
the changes were made as there was use of new transferhistory model from the api wrapper